src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2172,6 +2172,36 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_CreateMaskForFirstActiveElement:
+            {
+                assert(isRMW);
+                assert(HWIntrinsicInfo::IsExplicitMaskedOperation(intrin.id));
+
+                if (targetReg != op2Reg)
+                {
+                    assert(targetReg != op1Reg);
+                    GetEmitter()->emitIns_Mov(INS_sve_mov, emitTypeSize(node), targetReg, op2Reg, /* canSkip */ true);
+                }
+
+                GetEmitter()->emitIns_R_R(ins, emitSize, targetReg, op1Reg, INS_OPTS_SCALABLE_B);
+                break;
+            }
+
+            case NI_Sve_CreateMaskForNextActiveElement:
```

Comment:
For this one can you remove `HW_Flag_SpecialCodeGen` and then everything here will be done by common code?

Reply:
It uses `INS_sve_mov`. I could expand `emitIns_Mov` to handle it though.

Reply:
Is it possible to update the existing `isRMW` code to handle these intrinsics? It takes care of `mov` as well.

Reply:
Even if we cannot do it, the body of each case is idential and should be shared.

Reply:
It can, but would need to update emitIns_Mov to be aware of predicate registers and use INS_sve_mov for them.

Reply:
without `SpecialCodeGen` flag, where in the current code do we go, where we call `emitIns_Mov()`?